templates/registration/login.html

```diff
@@ -1,3 +1,4 @@
+{% load crispy_forms_filters %}
 {% extends "base.html" %}
```

Comment:
`{% extends "base.html" %}` always must be at first place.

`{% load crispy_forms_filters %}`
If you use this style, then replace {{ form.as_p }} to {{ form|crispy }} for making website more 🎀beautiful🎀 as readme said.
```suggestion
{% extends "base.html" %}
{% load crispy_forms_filters %}
```